feature/experimental/bgp/ate_tests/base_bgp_session_parameters/base_bgp_session_parameters_test.go

```diff
@@ -286,10 +286,8 @@ func TestEstablishAndDisconnect(t *testing.T) {
 	// Send Cease Notification from ATE to DUT
 	t.Log("Send Cease Notification from ATE to DUT")
 	ate.Actions().NewBGPPeerNotification().WithCode(6).WithSubCode(6).WithPeers(bgpPeer).Send(t)
-
-	// Verify BGP session state : ACTIVE
 	t.Log("Verify BGP session state : ACTIVE")
-	gnmi.Await(t, dut, nbrPath.SessionState().State(), time.Second*60, oc.Bgp_Neighbor_SessionState_ACTIVE)
+	gnmi.Await(t, dut, nbrPath.SessionState().State(), time.Second*20, oc.Bgp_Neighbor_SessionState_ACTIVE)
```

Comment:
@xw-g this actually reduced the await time. Are you okay with this?

Reply:
@sachendras to comment further.

@liulk feel free to merge when all your comments are resolved.

Reply:
updated time back to 60 secs